added start/start-dev commands

matt-oc opened the pull request.

matt-oc committed:
updated readme

updated readme

made logo direct link

added travis file

updated readme and added travis file

added messages section

added start/start-dev commands

added tagging and logging to readme

fixed typo

readme update

Json update

Json update

added CoC file

mcdonnelldean commented on package.json:
```diff
@@ -19,5 +19,9 @@
-  }
+  },
+  "scripts": {
+    "start-dev": "node srv/search-dev.js --seneca.options.tag=search --seneca.log.all",
```
Remove the options, these can be passed when running (the -- things)

mcdonnelldean commented on README.md:
```diff
@@ -9,13 +9,17 @@ system. Please see the [main repo][] for more details
+## Running
+npm run start
+npm run start-dev
+
```
Add a subsection on tagging and logging, you do this by doing

npm run start -- --your.options

The -- tells npm to forward options to the commands it is running

mcdonnelldean commented on README.md:
```diff
@@ -6,16 +6,41 @@ system. Please see the [main repo][] for more details
+## Tagging and Logging
+Options can be specified using '--'
+
+npm run start --your.options
```
remove

mcdonnelldean commented on README.md:
````diff
@@ -6,16 +6,41 @@ system. Please see the [main repo][] for more details
+
+```sh
+// tagging example
+npm run start --seneca.options.tag=npm
````
options are npm run start -- --your.options. You are missing a pair of --

matt-oc committed:
readme updated

matt-oc committed:
# This is a combination of 2 commits.
# The first commit's message is:

updated readme

# This is the 2nd commit message:

updated readme

matt-oc committed:
# This is a combination of 2 commits.
# The first commit's message is:

resolved conflicts

# This is the 2nd commit message:

added messages section

matt-oc committed:
# This is a combination of 2 commits.
# The first commit's message is:

resolved conflicts

# This is the 2nd commit message:

updated readme

matt-oc committed:
resolved conflicts

matt-oc committed:
resolved conflicts

matt-oc committed:
fixed conflicts
